php/stats.php

```diff
@@ -39,101 +41,121 @@
     $sth->execute([$user]);
     $row = $sth->fetch();
```

Comment:
We should pass `PDO::FETCH_ASSOC` as the first parameter, as we don't need the 0 based indexed columns anymore.